libs/lora-gateway/Makefile

```diff
@@ -0,0 +1,89 @@
+#
+# Copyright (C) 2018 OpenWrt.org
+#
+# This is free software, licensed under the GNU General Public License v2.
+# See /LICENSE for more information.
+#
+
+include $(TOPDIR)/rules.mk
+
+PKG_NAME:=lora-gateway
+PKG_VERSION:=5.0.1
+PKG_RELEASE:=1
+
+PKG_SOURCE:=v$(PKG_VERSION).tar.gz
+PKG_SOURCE_URL:=https://github.com/Lora-net/lora_gateway/archive/
+PKG_MD5SUM:=5d07f8471e1a67920787e3879afe0cb6
+PKG_MAINTAINER:=Xue Liu <liuxuenetmail@gmail.com>
+PKG_LICENSE_FILES:=LICENSE
+PKG_BUILD_DIR:=$(BUILD_DIR)/lora_gateway-$(PKG_VERSION)
+
+include $(INCLUDE_DIR)/package.mk
+
+define Package/libloragw
+  SECTION:=libs
+  CATEGORY:=Libraries
+  TITLE:=Driver/HAL library for Semtech SX1301
+  URL:=http://www.semtech.com/wireless-rf/lora.html
+  MENU:=1
+endef
+
+define Package/libloragw/description
+  Driver/HAL library for Semtech SX1301 multi-channel modem and 
+  SX1257/SX1255 RF transceivers.
+endef
+
+define Package/libloragw/config
+	source "$(SOURCE)/Config.in"
+endef
+
+define Package/lora-gateway-tests
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=LoRaWAN
+  TITLE:=Test programs for libloragw
+  DEPENDS:=+libloragw +librt
+endef
+
+define Package/lora-gateway-utils
+  SECTION:=net
+  CATEGORY:=Network
+  SUBMENU:=LoRaWAN
+  TITLE:=Utilities for libloragw
+  DEPENDS:=+libloragw
+endef
+
+TARGET_CFLAGS += $(FPIC) -O2 -Wall -Wextra -std=c99 -Iinc -I. -lm \
+		-DSPI_SPEED=$(CONFIG_SX1301_SPI_SPEED) \
+		-DSPI_DEV_PATH=\\\"\"$(CONFIG_SX1301_SPI_PATH)\\\"\"
+
+define Build/InstallDev
+	$(INSTALL_DIR) $(1)/usr/include/libloragw
+	$(CP) $(PKG_BUILD_DIR)/libloragw/inc/* $(1)/usr/include/libloragw
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(CP) $(PKG_BUILD_DIR)/libloragw/libloragw.a $(1)/usr/lib/
+endef
+
+define Package/libloragw/install
+	$(INSTALL_DIR) $(1)/usr/lib
+	$(INSTALL_BIN) $(PKG_BUILD_DIR)/libloragw/libloragw.a $(1)/usr/lib/
```

Comment:
it's only available as a static library?  how is that intended to be used?  Is there any point in installing here as well as in dev?

Reply:
Yes. It is originally only avaibale as a static library. It is used for packet_forwarder which is used to build a LoRaWAN gateway. https://github.com/Lora-net/packet_forwarder,